pkg/aws/aws.go

```diff
@@ -60,6 +63,14 @@ const (
 	AssumeRoleDuration        = "assumeRoleDuration"
 )
 
+var _ stscreds.TokenFetcher = (*staticToken)(nil)
+
+type staticToken string
+
+func (f staticToken) FetchToken(ctx credentials.Context) ([]byte, error) {
```

Comment:
[nit] some comments will help w understanding: Implements `TokenFetcher` interface from ... for static web identity token

Reply:
Fixed